projects/go-lib/src/lib/components/go-button/go-button.component.ts

```diff
@@ -18,22 +17,28 @@ export class GoButtonComponent {
 
   constructor() {}
 
-  public clicked() : void {
-    if (this.isProcessing || this.buttonDisabled) { return; }
-
+  public clicked(): void {
     this.isProcessing = this.useLoader;
     this.handleClick.emit(this.isProcessing);
   }
   
-  public reset() : void {
+  public reset(): void {
     this.isProcessing = false;
   }
 
-  public classList() : any {
+  public classObject(): object {
+    // 'alert' as a variant is depreciated and
+    // will be removed in a later version
```

Comment:
I like this concept, allows us to communicate deprecated changes ahead of time. :+1: